instance/models/openedx_appserver.py

```diff
@@ -361,13 +361,30 @@ def smtp_relay_settings(self):
         else:
             return None
 
+    @property
+    def organization_users(self):
+        """
+        :return: Github usernames of the organization this instance owned by.
+        """
+        users = []
+        organization = self.instance.ref.owner
+        if organization:
+            org_github_handle = organization.github_handle
+            usernames = get_username_list_from_team(org_github_handle)
```

Comment:
@ahmedaljazzar  get_username_list_from_team is obsolete, we should instead use the users that are registered on ocim under the organization that owns the instance.

So something like:

`python
UserProfile.objects.filter(organization=organization).values_list('github_username', flat=True)
`

Reply:
We should also remove the function `get_username_list_from_team` together with `github_admin_username_list`